master/buildbot/test/fake/fakedb.py

```diff
@@ -90,6 +91,10 @@ def __init__(self, **kwargs):
         for k, v in iteritems(self.values):
```

Comment:
I'll try and see is it worth removing: if there will thousands of places to changes, it's not worth it, but if almost everywhere unicode string are already passed, then it may worth it.